Mobile/ShopLens/MigrationsConsole/ShopLensRandomUserGenerator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using RandomNameGenerator;
+using ShopLensWeb;
+using System;
+using System.Security.Cryptography;
+
+namespace ShopLens.Droid.Helpers
+{
+    public class ShopLensRandomUserGenerator
```

Comment:
I tried adding the ShopLens.Droid project as a dependancy to the MigrationsConsole project but then ShopLensContext class overlapped, because there were two classes added to MigrationsConsole with the same name ShopLensContext, so that's why I created a duplicate random name generator and put it in MigrationsConsole.

Reply:
If you look closely, I didn't copy-paste any models to the MigrationsConsole project. I just added the links to the files. Maybe it would work for you if you did the same thing with this class?

I think the original should be in ShopLensWeb, but a link to it can be in MigrationsConsole.